packages/app/src/renderer/utils/form-validators.test.tsx

```diff
@@ -168,5 +168,14 @@ describe("Form Validators", () => {
         ).toBeInTheDocument()
       })
     })
+    test("should pass as valid when password value contains at least one of &'()*+,-./:;<=>?@[]^_`{|}~ ", () => {
```

Comment:
🟡
I think that will be better if each listed character should be a part of a test instead of a description.

I suggest to refactor this to `describe` + [test.each](https://jestjs.io/docs/api#testeachtablename-fn-timeout).